.github/workflows/create_release.yml

```diff
@@ -254,11 +278,37 @@ jobs:
           verbose: true     
           print-hash: true  
 
-  test_source_dist:
-    name: test source distribution
-    needs: [publish_preview_build_to_pypi_weekly]
-    if: github.event_name != 'pull_request' || contains(github.event.pull_request.labels.*.name, 'run release CIs')
-    uses: ./.github/workflows/preview_source_dist_test.yml
-    with:
-      os: "macos"
+  publish_release_build_to_pypi:
+    name: Publish release build to pypi
+    runs-on: ubuntu-latest
+    needs: [check_for_publish_release_build_to_pypi]
+
+    environment:
+      name: pypi-release # TODO: has to be created and configured for rel-1.19.0
```

Comment:
I think we should configure the approval group properly to guard this from being mis-triggered

Reply:
1) the environment name mentioned here does not exist so far, 2) trusted publishing was not configured at pypi for onnx-release

=> probably we have to try that out for the 1.18rc1 candidate publishing to testpypi